CONTRIBUTING.adoc

```diff
@@ -0,0 +1,46 @@
+= Contributing
+// settings:
+:idprefix:
+:idseparator: -
+:source-language: ruby
+:language: {source-language}
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+// URIs:
+:uri-repo: https://github.com/asciidoctor/asciidoctor
+:uri-issues: {uri-repo}/issues
+:uri-squash-help: https://chris.beams.io/posts/git-commit/#seven-rules
+:uri-gist: https://gist.github.com
+
+== License Agreement
+
+By contributing changes to this repository, you agree to license your contributions under the MIT license.
```

Comment:
Licence is Apache 2. Don't ask me why :man_shrugging:

Reply:
I wonder if I should do something like: 

`Copyright [yyyy] [name of copyright owner]

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.`

Reply:
I think just referencing the licence by name is enough, maybe for ease, link the filename in the root of the repo.